.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install the dependencies, run tests and lint every push
+
+name: Build Java
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: ['8', '11', '17']
```

Comment:
Why 11 and 17?

Reply:
LTS versions

Reply:
But we only target 8

Reply:
Really? that's not great, but we can remove them if so

Reply:
It would require an effort to make it build with future versions. Not a huge effort, but it'd have to be prioritized.